File: Solutions/2659/c++/program.cpp
// Copy from Quera

#include <iostream>

using namespace std;

int main()
{
    int n;
    cin >> n;

    char s[n];
    for (int i = 0; i < n; i++)
        cin >> s[i];

    char t[n];
    for (int i = 0; i < n; i++)
        cin >> t[i];

    int ans = 0;
    for (int i = 0; i < n; i++)
        if (s[i] != t[i])
            ans++;

    cout << ans << '\n';
    return 0;
}

File: Solutions/2659/java/program.java
// Copy from Quera

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
		char[] s = scanner.next().toCharArray();
		char[] t = scanner.next().toCharArray();

		int ans = 0;
		for (int i = 0; i < n; i++)
			if (s[i] != t[i]) 
				ans++;

		System.out.println(ans);
    }
}
